### DISCUSSION TITLE
Will tools like dxvk be installed automatically for every program?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2322] [Q&A]
If I install a program,will tools like dxvk and fonts be installed automatically?
And can I select which tools will be installed for every program (so I don't have to do it manually)?

**Comment by plata** [comment]
No. The install script defines which tools will be installed.

**Reply by ghost** [reply]
What if you add an already installed program?

**Reply by plata** [reply]
You can install Verbs in the containers tab.
